src/lib/controls/Checkbox.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte';
+
+	export let id: string;
+	export let checked: any;
+	export let label: string = '';
+	export let sideLabel: boolean = false;
+	export let labelAfter: boolean = false;
+	export let disabled: boolean = false;
+
+	const dispatch = createEventDispatcher();
+</script>
+
+<div class:hstack={sideLabel}>
+	{#if !labelAfter}
+	<label for={id}>
+		{label}
+		<slot />
+	</label>
+	{/if}
+	<input {id} type="checkbox" bind:checked {disabled}
+		on:change={() => dispatch('change')}/>
```

Comment:
Doing this will pass the event along:
```suggestion
		on:change />
```